ieeetran/footer.tex

```diff
@@ -5,10 +5,27 @@
 \printbibliography
 \fi
 
+\appendices
```

Comment:
So when you look at the `main.tex` file that comes with the `IEEEtran.cls`  package as a template to copy from, the call `\appendices` is made before include either the appendix, biography, or acknowledgments. I'm not sure what the best way to handle including these three things conditionally are after it, so I included it regardless of whether or not they were added.